srcpkgs/refind/files/refind-kernel-hook.conf

```diff
@@ -18,5 +18,11 @@ UPDATE_REFIND_CONF=0
 # /efi/EFI/BOOT/refind.conf
 REFIND_CONF=/boot/EFI/refind/refind.conf
 
-# addition kernel cmdline
+# Set the label of the Void Linux filesystem
+REFIND_LABEL="Void Linux"
```

Comment:
Leave the value commented out by default, and make the leading comment more descriptive:
```suggestion
# Set a custom label for Void boot entries
#REFIND_LABEL="Void Linux"
```